index.html

```diff
@@ -15,29 +15,35 @@
     <div class="container">
       <header>
         <div class="logo-placeholder">
-          <p class="logo">M.D</p>
+          <a href="#" class="logo">md</a>
         </div>
         <nav class="nav-links">
-          <img class="menu-btn" src="/images/menu-btn.svg" alt="Menu Button" />
+          <div class="menu-btn"><i class="bx bx-menu"></i></div>
         </nav>
       </header>
-      <main>
+      <section class="main">
         <h1 class="title">I'm Derrick. Nice to see you!</h1>
         <p class="about-me">
           I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hesitate to contact
           me.
         </p>
-      </main>
-      <footer>
+      </section>
+      <section class="footer">
         <h4 class="social-title">LET'S CONNECT</h4>
-        <div class="social-icons">
-          <a href="#"><i class="bx bxl-twitter"></i></a>
-          <a href="#"><i class="bx bxl-linkedin"></i></a>
-          <a href="#"><i class="bx bxl-github"></i></a>
-        </div>
```

Comment:
- [ ] As requested by the previous reviewer, kindly add a background image to this section to match the design template.
*Your project*![ccc](https://user-images.githubusercontent.com/94551484/185498189-6ff8e31a-0fa2-40c6-a78a-836d8d446e03.png)
*Design template*![aaa](https://user-images.githubusercontent.com/94551484/185498217-ae5e1d9f-6f97-40dc-9748-f4cbc026e905.png)

- [ ] Kindly change your alignments to match the design template. Please note that your project design should look exactly like the Figma template you chose.